libs/scripts/scripts.js

```diff
@@ -117,5 +117,7 @@ const config = {
 (async function loadPage() {
   setConfig(config);
   await loadArea();
-  loadDelayed();
+  loadDelayed().then((interlinksMod) => {
+    if (interlinksMod) interlinksMod.default('/keywords.json');
```

Comment:
Can we do this inside of `loadDelayed` instead?  I'd prefer to keep this as clean as possible.